frontend/src/components/Results.js

```diff
@@ -2,17 +2,70 @@ import React, { Component } from 'react'
 import { Link } from 'react-router-dom'
 import { connect } from 'react-redux'
 import Card from './Card'
+import {
+  TabContent,
+  TabPane,
+  Nav,
+  NavItem,
+  NavLink,
+  Button,
+  CardTitle,
+  CardText,
+  Row,
+  Col
+} from 'reactstrap'
+import classnames from 'classnames'
 
 const mapStateToProps = state => ({
   results: state.results.games
 })
 
 class Results extends Component {
-  buildCards = this.props.results.map(c => <Card game={c} />)
+  constructor(props) {
+    super(props)
+
+    this.toggle = this.toggle.bind(this)
+    this.state = {
+      activeTab: '1'
+    }
+  }
+
+  toggle(tab) {

```

Comment:
If you change this to `toggle = tab => {...}` you don't need to do the bind in the constructor, which makes the code cleaner.